Okay, so what do we do about the missing API...i'm guessing this was a breaking change missed a while back.

@JeremyKuhne any of this look familiar from mid-.net 5 previews?

```
src\WebSdk\Publish\Tasks\Tasks\MsDeploy\GetPassword.cs(46,36): error CA1416: (NETCORE_ENGINEERING_TELEMETRY=Build) 'ProtectedData.Unprotect(byte[], byte[]?, DataProtectionScope)' is supported on 'windows'

src\WebSdk\Publish\Tasks\Tasks\MsDeploy\GetPassword.cs(46,110): error CA1416: (NETCORE_ENGINEERING_TELEMETRY=Build) 'DataProtectionScope.CurrentUser' is supported on 'windows'

```

--- a/src/WebSdk/Publish/Tasks/Tasks/MsDeploy/GetPassword.cs
+++ b/src/WebSdk/Publish/Tasks/Tasks/MsDeploy/GetPassword.cs
@@ -43,7 +43,9 @@ namespace Microsoft.NET.Sdk.Publish.Tasks.MsDeploy
             string plainPWD = "";
             try
             {
+#pragma warning disable CA1416 // This functionality is only expected to work in Windows.
                 Byte[] uncrypted = System.Security.Cryptography.ProtectedData.Unprotect(encryptedData, null, System.Security.Cryptography.DataProtectionScope.CurrentUser);
+#pragma warning restore CA1416 // This functionality is only expected to work in Windows.
                 plainPWD = Encoding.Unicode.GetString(uncrypted);
             }
             catch

